### PR TITLE
Removed Addons Connect button in Edit Employee Form in BO

### LINKED ISSUE #25468: [BO] - An exception thrown when you try to access "edit your profile"
#### Describe the bug

When you try to edit your profile as an admin, an exception will be thrown

#### Expected behavior

No exception

#### Steps to Reproduce

1. **BO** > **CONFIGURE** > **Advanced Parameters** > **Team**
2. Try to edit your profile (employee)
3. See error => an exception is thrown
4. Try to edit another admin profile or any employee profile => no error
5. Try to edit your profile from **employee-dropdown** in the header (see attached screenshot n°1)
6. See error => an exception is thrown (see attached screenshot n°2)

**Screenshot n°1**

image

**Screenshot n°2**

image

#### Additional information

* PrestaShop version: (only on) develop
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Removed Addons Connect button in Edit Employee Form in BO
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | Fixes #25468
| How to test?      | Cf. #25468

**:notebook: BC Breaks :**
* In `src/Core/Employee/Access/EmployeeFormAccessCheckerInterface.php`, remove the declared method `public function canAccessAddonsConnect();`
* In `src/Adapter/Employee/EmployeeFormAccessChecker.php`:
  * the 3rd parameter has been removed `PrestaShopBundle\Entity\Repository\TabRepository $tabRepository`
  * the method `canAccessAddonsConnect()` has been removed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25559)

### CODE REVIEW

**Comment by PierreRambaud** [comment]
Already did it here #25245 in case you want to compare, I will rebase my PR after this one is merged.
(I'm waiting for welcome module release to make CI happy)

**Comment by Progi1984** [comment]
Thank you @PierreRambaud. I have brought some comments to your PR #25245.

**Comment by Progi1984** [comment]
I finally reopen : @PierreRambaud will rebase his PR.

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by florine2623** [comment]
Hello @Progi1984 ,

Thanks for the PR, it works well :)

**Review by PululuK — APPROVED** [review]

**Comment by PululuK** [comment]
Thanks @Progi1984

**Pull request merged**
